opentdf-dev.yaml

```
@@ -66,7 +67,31 @@ server:
       #
       #  [matchers]
       #  m = g(r.sub, p.sub) && globOrRegexMatch(r.res, p.res) && globOrRegexMatch(r.act, p.act) && globOrRegexMatch(r.obj, p.obj)
-
+  cors:
+    enabled: false
+    # '*' to allow any origin or a specific domain like 'https://yourdomain.com'
+    allowedorigins: '*'
```

Shouldn't these be lower camelCase?
```suggestion
    allowedOrigins: '*'
```

---

Viper does not allow camelcase https://github.com/opentdf/platform/issues/745